include/mercury/http/http_client.h

```diff
@@ -21,17 +21,19 @@
 
 #include "mercury/http/http_client_interface.h"
 
-
 namespace Mercury {
 namespace HTTP {
 
+#define DEFAULT_HTTP_CLIENT_TIMEOUT    15 // seconds
```

Comment:
```suggestion
const unsigned int DEFAULT_HTTP_CLIENT_TIMEOUT = 15;  // seconds
```

Also, is 15 seconds too long?